src/main/java/oncall/validation/IntegerValidator.java

```diff
@@ -0,0 +1,71 @@
+package oncall.validation;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+import oncall.util.ExceptionUtil;
+import oncall.util.IntegerUtil;
+
+public final class IntegerValidator {
+
+    public static void validateInteger(final String string) {
+        if (!IntegerUtil.isInteger(string)) {
+            ExceptionUtil.throwInvalidValueException();
+        }
+    }
+
+    public static void validateInteger(final String... strings) {
+        Arrays.stream(strings)
+                .forEach(string -> validateInteger(string));
+    }
+
+    public static void validateInteger(List<String> separated) {
+        boolean result = separated
+                .stream()
+                .allMatch(value -> IntegerUtil.isInteger(value));
+
+        if (!result) {
+            ExceptionUtil.throwInvalidValueException();
+        }
```

Comment:
해당 로직이 제공하고자 하는 "`List` 내부의 값들이 숫자인지 검증" 기능은 이미 해당 클래스에 구현되어 있는 `validateInteger()` 메서드와 궤를 같이 한다고 생각해요. 해당 기능을 잘 구현해 두신 만큼, 코드의 재활용을 통해 중복되는 로직을 줄일 수 있을 것 같습니다!
```suggestion
        separated.forEach(value -> validateInteger(value));
```

Reply:
오오 감사합니다 👍

Reply:
사실 이 방법을 쓰면 더더욱 IntegerUtil이 필요한가...라는 생각이 드네요...ㅎ